precompiles/EcPairing.yul

```diff
@@ -549,30 +667,57 @@ object "EcPairing" {
 
             // FP2 ARITHMETHICS
 
+            /// @notice Compute the sum of two Fp2 elements.
+            /// @dev Algorithm 5 in: https://eprint.iacr.org/2010/354.pdf.
+            /// @param a00, a01 The coefficients of the A element to sum.
+            /// @param b00, b01 The coefficients of the B element to sum.
+            /// @return c00, c01 The coefficients of the elemtnt C = A + B.
```

Comment:
I think that all the functions for arithmetics in the tower field extension have a typo in the word `element` in the return tag.

Reply:
Yes! Let me add that we should also change all the `Compute` with `Computes`, and there are some singular cases in which we'd use `an Fpx element` instead of `a Fpx element`.